docker-compose.yml.example

```diff
@@ -28,7 +28,7 @@ services:
           - backend-django-tosti
 
     tosti-django:
-        image: "larsvanrhijn/tosti:latest"
+        image: "docker.pkg.github.com/kioui/tosti/tosti:latest"
```

Comment:
In the docker.yaml file, you reference it as `docker.pkg.github.com/${GITHUB_REPOSITORY}/${GITHUB_REPOSITORY##*/}:latest`, but shouldn't it be the repository owner for the first one

Reply:
Nope, check the latest docker image in the repo and the latest succeeded docker actions build. GITHUB_REPOSITORY outputs KiOui/TOSTI

Reply:
And ${GITHUB_REPOSITORY##*/} outputs TOSTI